enrichments/trace/internal/elastic/span.go

```diff
@@ -434,6 +462,43 @@ func (s *spanEventEnrichmentContext) enrich(
 	if cfg.ProcessorEvent.Enabled && s.exception {
 		se.Attributes().PutStr(AttributeProcessorEvent, "error")
 	}
+	if s.exceptionType == "" && s.exceptionMessage == "" {
+		// Span event does not represent an exception
+		se.Attributes().PutStr(AttributeEventKind, "event")
+		se.Attributes().PutStr(AttributeMessage, se.Name())
+		return
+	}
+
+	// Span event represents exception
+	if cfg.ErrorID.Enabled {
+		if id, err := newUniqueID(); err == nil {
+			se.Attributes().PutStr(AttributeErrorID, id)
+		}
+	}
+	if cfg.ErrorExceptionType.Enabled && s.exceptionType != "" {
+		se.Attributes().PutStr(AttributeErrorExceptionType, s.exceptionType)
+	}
+	if cfg.ErrorExceptionMessage.Enabled {
+		se.Attributes().PutStr(AttributeErrorExceptionMessage, s.exceptionMessage)
+	}
+	if cfg.ErrorExceptionHandled.Enabled {
+		se.Attributes().PutBool(AttributeErrorExceptionHandled, !s.exceptionEscaped)
+	}
+	if cfg.ErrorStacktrace.Enabled {
+		// TODO (lahsivjar): Where to parse stacktraces?
```

Comment:
[**For reviewers**] In apm-data, we parse the stacktrace (only for Java) should we be doing this here to? If yes, then how should we add the parsed stacktrace?  -- I can only think of array of string where each element is a string encoded JSON representing the format expected by Elastic UI.

Reply:
I think we can leave this out for now.

Alternatively, we can parse the stack trace but only use it for the error grouping hash.

Reply:
> Alternatively, we can parse the stack trace but only use it for the error grouping hash

Sounds good. Since we are not going to parse the stacktrace, I will work on this as a follow-up.